go/auth/token.go

```diff
@@ -17,12 +19,15 @@ import (
 )
 
 const (
-	TokenType           = "auth"
-	CurrentTokenVersion = 1
+	TokenType             = "auth"
+	CurrentTokenVersion   = 2
+	ChallengeLengthBytes  = 32
+	ChallengeLengthString = ChallengeLengthBytes*2 // we use hex encoding
 )
 
 type TokenAuth struct {
-	Server string `json:"server"`
+	Server    string `json:"server"`
+	Challenge string `json:"challenge"`
```

Comment:
Can you call this field `session` on the wire so it matches the name in `libkb/kbsig`?  It's fine to call it `Challenge` in the code.